modules/vstudio/tests/cs2005/test_documentation_file.lua

```diff
@@ -0,0 +1,68 @@
+--
+-- tests/actions/vstudio/cs2005/test_debug_props.lua
+-- Test debugging and optimization flags block of a Visual Studio 2005+ C# project.
+-- Copyright (c) 2012-2013 Jason Perkins and the Premake project
+--
+	local p = premake
+	local suite = test.declare("vstudio_cs2005_documentation_file")
+	local dn2005 = p.vstudio.dotnetbase
+--
+-- Setup
+--
+
+	local wks, prj
+
+--
+-- Setup and teardown
+--
+	function suite.setup()
+		p.action.set("vs2010")
+		wks = test.createWorkspace()
+		configurations { "Debug", "Release" }
+		-- project "documentationFile"
+		language "C#"
+		targetdir("test\\targetDir")
+	end
+
+	local function setConfig()
+		local cfg = test.getconfig(prj, "Debug")
+		dn2005.documentationFile(cfg);
+	end
+
+
+
+	local function prepare()
+		prj = test.getproject(wks, 1)
+	end
+
+	local function prepareEmpty()
+		prepare()
+		documentationFile ""
+		setConfig()
+	end
+
+	local function prepareDir()
```

Comment:
Can we add a test for when it is not set at all?

Reply:
Sure will leave comment when done

Reply:
i implemented the comments.